### DISCUSSION TITLE
Update image with docker hub webhooks

### DISCUSSION THREAD

**Discussion by ValerioNeriGit** [containrrr/watchtower#958] [Q&A]
Hi, 
Is it possible to update the images (or a single one) based on the docker hub webhook?  
I'd like to update the container as soon as possible.

Thanks

**Comment by simonaronsson** [comment]
Not currently, but it sure would be a cool feature!

EDIT: I guess you **could** accomplish something similar by exposing the HTTP API, but given that watchtower has direct access to the docker.sock, I wouldn't recommend it. Instead, you should probably place some small rest api in front of it that only allows this specific action.
